modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/collectors/OpenEventCollector.java

```diff
@@ -54,21 +54,15 @@ public class OpenEventCollector extends RuntimeStatisticCollector {
 	 */
 	public static Integer reportEntryEvent(MessageContext messageContext, String componentName,
 	                                       AspectConfiguration aspectConfiguration, ComponentType componentType) {
-		boolean isCollectingStatistics = (aspectConfiguration != null && aspectConfiguration.isStatisticsEnable());
-
 		// Enable statistics, if user enabled for all artifacts
-		if (!isCollectingStatistics) {
-			isCollectingStatistics = isCollectingStatistics || RuntimeStatisticCollector.isCollectingAllStatistics();
-		}
+		boolean isCollectingStatistics = (aspectConfiguration != null && aspectConfiguration.isStatisticsEnable())
+				|| RuntimeStatisticCollector.isCollectingAllStatistics();
 
-		boolean isCollectingTracing = false;
-		if (isCollectingProperties() || isCollectingPayloads() || isCollectingVariables()) {
-			isCollectingTracing = (aspectConfiguration != null && aspectConfiguration.isTracingEnabled());
-		}
+        boolean isCollectingTracing = (isCollectingProperties() || isCollectingPayloads() || isCollectingVariables())
```

Comment:
Shall we fix this bad formatting?

Reply:
Fixed with 9ad2e07ac3e53d9672613a1fc8693bdd5a7a6644